components/src/core/components/SidePanel/MainMenu.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <!-- Search -->
-  <div class="oxd-main-menu">
+  <div class="oxd-main-menu" style="flex: none;">
```

Comment:
Any special reason to keep inline styles

Reply:
not really. added as a class.